examples/auto-scaling-groups/destroy.pp

```diff
@@ -0,0 +1,21 @@
+cloudwatch_alarm { ['AddCapacity', 'RemoveCapacity']:
+  ensure => absent,
+  region => 'sa-east-1',
+} ~>
+
+ec2_scalingpolicy { ['scaleout', 'scalein']:
```

Comment:
I'm not so sure about the `ec2` prefixes for ASG resources... given these are not part of the EC2 API, and we don't use the EC2 client to configure them, but rather the ASG client instead.